### PR TITLE
Add example code to docstrings for resize

### PR DESCRIPTION
**Pull request by trbromley**
Adding examples of function use in small code snippets can really help the reader get a quick understanding of the function. We would like to do this for functions in the applications layer, and this PR provides code snippets for max-clique based functions in `graph.resize`.

### CODE REVIEW

**Comment by trbromley** [comment]
This PR is left as WIP while we agree on the styling details in the sister PR https://github.com/XanaduAI/strawberryfields/pull/177. Once done, we'll update here and mark as ready for review.

File: strawberryfields/apps/graph/resize.py [hunk 1]
```diff
@@ -256,6 +256,22 @@ def clique_grow(clique: list, graph: nx.Graph, node_select: str = "uniform") ->
...
+        >>> import networkx as nx
+        >>> graph = nx.complete_graph(10)
+        >>> clique = [0, 1, 2, 3, 4]
+        >>> resize.clique_grow(clique, graph)
+        [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+        >>> graph = nx.lollipop_graph(5, 1)
+        >>> graph.remove_edge(3, 4)
+        >>> clique = [0, 1, 2]
+        >>> resize.clique_grow(clique, graph, node_select="degree")
+        [0, 1, 2, 4]
```
Comment (josh146) [inline strawberryfields/apps/graph/resize.py]: Note that the use of `>>>` negates the need to use `.. code-block::` and indent.

**Comment by trbromley** [comment]
Updated based on feedback from https://github.com/XanaduAI/strawberryfields/pull/177.

**Comment by trbromley** [comment]
@ixfoduap @josh146 - going to merge this in tomorrow morning! 😁

**Pull request merged**